kafka/config.go

```diff
@@ -45,6 +46,8 @@ type (
 		// If this is empty, we will get the broker list using the NameResolver
 		// TODO (gteo): remove this and rely on a NameResolver as single source for broker IP
 		BrokerList []string
+		// RetryDelay, msg diliver delay duration for retryTopic;
+		RetryDelay time.Duration
```

Comment:
I think this should be called `Delay` instead of `RetryDelay` since the `Topic` object has no notion of type.